spec/ruby-enum/enum_spec.rb

```diff
@@ -188,6 +188,14 @@ class EmptyEnums
         expect(subject::PINK).to eq 'pink'
       end
```

Comment:
let's add tests for the various levels here

Reply:
I added another test for the `SecondSubclass.values` value here. Let me know if this is not what you had in mind.